ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -20,20 +20,11 @@
 
 import biz.ganttproject.core.model.task.TaskDefaultColumn;
 import biz.ganttproject.core.option.BooleanOption;
-import com.google.common.base.Charsets;
-import com.google.common.base.Joiner;
-import com.google.common.base.Preconditions;
-import com.google.common.base.Predicate;
-import com.google.common.base.Strings;
+import com.google.common.base.*;
```

Comment:
Please keep explicit imports. There must be a setting in your IDE which specifies how many imports should be replaces with *. If you use IDEA then it is Editor->Code Style->Java->Class count to use import with *